.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Mon Aug 03 12:41:32 EEST 2020
+gradle.version=6.5
```

Comment:
please remove all unnecessary files from GitHub
In this case try to follow this instruction to remove them from PR
https://stackoverflow.com/a/14797830